crates/sdk/src/tx.rs

```diff
@@ -183,17 +183,15 @@ pub fn dump_tx<IO: Io>(io: &IO, args: &args::Tx, tx: Tx) {
 /// to it.
 #[allow(clippy::too_many_arguments)]
 pub async fn prepare_tx<C: crate::queries::Client + Sync>(
-    client: &C,
+    _client: &C,
```

Comment:
this argument should also be removed

Reply:
I don't know why this comment was marked as resolved. I would still like to have this argument removed. Especially since it removes the need for the generic, which should simplify other parts of the code base.